### ISSUE TITLE
No data in the folder

### ISSUE THREAD

**Issue by imhuaweizhu** [waymo-research/waymo-open-dataset#638]
I followed the link provided by Waymo, and downloaded the V1.2.0 Motion dataset, it only has 6 Bytes, basically empty. `https://console.cloud.google.com/storage/browser/waymo_open_dataset_motion_v_1_2_0;tab=objects?prefix=&forceOnObjectsSortingFiltering=false`

Am I missing sth?

**Comment by scott-ettinger** [comment]
Sorry for the trouble on this.  There are two directories in the link you shared.  One is named "uncompressed", the other is named "uncompressed_$folder$" and is an artifact of the GCP file system.  The "uncompressed" directory contains the dataset while "uncompressed_$folder$" is only 6 bytes.  

Could you check to see if you tried to download the correct folder? 

I will remove the extra "uncompressed_$folder$" directory.

**Comment by imhuaweizhu** [comment]
I tried that one too, I tried to download the data with this command  `gsutil -m cp -r "gs://waymo_open_dataset_motion_v_1_2_0/uncompressed" .` in my terminal, and I got the following error.

```
Unknown option: m
No command was given.

Choose one of -b, -d, -e, or -r to do something.
Try `/usr/bin/gsutil --help' for more information.
```

**Comment by scott-ettinger** [comment]
I tried the following command on my system and it works: 
gsutil -m cp -r "gs://waymo_open_dataset_motion_v_1_2_0/uncompressed" .

It looks like your error is around the -m flag not being supported.
Could you check is to see which version of gsutil you are using by running: 
```
gsutil version 
```
Could you also try running
```
gsutil help options
```
To see if it lists the -m option? 

Once this works, you may want to download only the format you plan to use.  If you plan to use scenario protos, use: 
gsutil -m cp -r "gs://waymo_open_dataset_motion_v_1_2_0/uncompressed/scenario" .

Note that the optional lidar data is in a separate directory.

The only other thing I would try is removing the quotation marks.  
Please let us know if you are still having trouble.

**Comment by imhuaweizhu** [comment]
![Selection_004](https://user-images.githubusercontent.com/34038625/233249596-93598144-b24d-4cc5-ae65-c340f7f07e0a.png)
there is no `-m` option

**Comment by imhuaweizhu** [comment]
@scott-ettinger Could it be the wrong version of `gutils` is installed?

**Comment by scott-ettinger** [comment]
Yes, it looks like you may have installed a different utility.  You will need to install the google cloud CLI which includes the gsutil command line utility.  You can find instructions here: https://cloud.google.com/sdk/docs/install

**Comment by shreyas1230** [comment]
I have a similar question but regarding the [waymo_open_dataset_v_2_0_0](https://console.cloud.google.com/storage/browser/waymo_open_dataset_v_2_0_0/validation/camera_segmentation) folder. Most of the `camera_segmentation` parquet files in the validation set and a few in the training set have size 10KB and when I try to open them and view the contents as a Dask dataframe, the number of rows in the dataframe is 0. After going through all the segmentation files in the validation directory, it seems like there are only around 20 nonempty files.

In contrast, all the camera images in the validation set are present. Looking through the images for one of the parquet files (`validation/camera_image/10203656353524179475_7625_000_7645_000.parquet`), it seems like the corresponding segmentation labels should be present since there are quite a few cars in view. Is the absence of segmentation label data for most of these sequences expected?

**Comment by trand2k** [comment]
I use this command for download waymo data 

gcloud storage cp -r gs://waymo_open_dataset_motion_v_1_2_0/uncompressed/scenario .

hope this can help

**Comment by thuongtrannc** [comment]
> I use this command for download waymo data
> 
> gcloud storage cp -r gs://waymo_open_dataset_motion_v_1_2_0/uncompressed/scenario .
> 
> hope this can help

It worked for me. Thanks alot @trand2k
